### ISSUE TITLE
Lumo Dark Theme for Vaadin 13

### ISSUE THREAD

**Issue by kyberorg** [appreciated/vaadin-app-layout#270]
**Describe the bug**
Seems like Lumo.DARK is broken (all elements are white, except Grid)
But Material.DARK works fine.

**To Reproduce**

1. Add `@Theme(value = Lumo.class, variant=Lumo.DARK)` to class, which extends `AppLayoutRouterLayout`
2. Run application

**Expected behavior**
All UI in Dark colours

**Screenshots**
Lumo.DARK
![Screenshot_20190507_125732](https://user-images.githubusercontent.com/4687253/57291782-c9a03280-70c8-11e9-82af-5caaccca06dc.png)

Same with Material.DARK
![Screenshot_20190507_125942](https://user-images.githubusercontent.com/4687253/57291814-dcb30280-70c8-11e9-8131-1b922735138e.png)

** Additional Information (please complete the following information):**
 - OS: Archlinux
 - Browser firefox, chrome, safari
 - Version 3.0.0.beta5

**Comment by kyberorg** [comment]
Workaround from examples
```
/**
     * Using the @Theme Annotation to set the Dark Theme causes issues with shadows which will appear in
     * the wrong color making them seemingly invisible. Instead do it the following way as long as the issue is not
     * solved (https://github.com/vaadin/flow/issues/4765)
     */
    @Override
    protected void onAttach(AttachEvent attachEvent) {
        super.onAttach(attachEvent);
        getUI().get().getPage().executeJavaScript("document.documentElement.setAttribute(\"theme\",\"dark\")");
    }```

**Comment by appreciated** [comment]
Just had a look into this, [Vaadin fixed this Issue for Flow 1.5.0.alpha1](https://github.com/vaadin/flow/issues/4765), which is shipped with Vaadin 14.
